### DISCUSSION TITLE
Change landing page after login

### DISCUSSION THREAD

**Discussion by wfillmer** [snipe/snipe-it#10273] [Q&A]
The default landing page in my install for users that are not admin is http://snipeit.local/account/view-assets

However, we have a generic guest account & group that cannot make any changes to assets, guests can only login and look at the database. Since the guest user will never have any asset checked out or assigned to them there is no value on the view-assets page. I would like the default for the guest account to fall on the http://snipeit.local/hardware page and see the list of assets after logging in..

Does anyone know which file I need to edit to change this?

**Comment by mihaiolimpiu** [comment]
I second this, due to the way we have our login systems implemented, all techs need to login with accounts that don't have any devices on them so would be best they go directly to the hardware page.
